src/main/kotlin/org/dif/common/Algorithms.kt

```diff
@@ -1,9 +1,14 @@
 package org.dif.common
 
+/**
+ * Encryption algorithms marker.
+ */
+sealed interface CryptAlg
```

Comment:
Didn't know sealed interfaces existed, just classes!! It always annoyed me I couldn't use sealed classes with enums, thank you for this lesson! ;p